avatars/animationHelpers.js

```diff
@@ -939,30 +949,44 @@ export const _applyAnimation = (avatar, now, moveFactors) => {
           if (!isPosition) {
             const src2 = useAnimation.interpolants[k];
             const v2 = src2.evaluate(t2);
+            localQuaternion2.fromArray(v2);
 
-            const idleAnimation = _getIdleAnimation('walk');
-            const t3 = 0;
-            const src3 = idleAnimation.interpolants[k];
-            const v3 = src3.evaluate(t3);
+            if (moveFactors.crouchFactor === 0) {
+              if (isCombo) {
+                // do nothing: localQuaternion2 already pure combo animation
```

Comment:
I really do not recommend depending on locals like this. Locals are reused and constantly change their meaning, so this comment will be irrelevant and wrong very quickly.

If the local is meaningful enough to be talked about in a comment, it should be given an actual name like:

```
const headQuaternion = localQuaternion2;
```